piezo_web_app/PiezoWebApp/src/services/spark_job/validation/manifest_populator.py

```diff
@@ -91,7 +92,15 @@ def _default_spark_application_manifest(self):
                                 "key": "accessKey"},
                             "AWS_SECRET_ACCESS_KEY": {
                                 "name": self._secret_name,
-                                "key": "secretKey"}}}}}
+                                "key": "secretKey"}}
+                    },
+                    "monitoring": {
+                        "exposeDriverMetrics": True,
+                        "exposeExecutorMetrics": True,
+                        "prometheus": {
+                            "jmxExporterJar": "/prometheus/jmx_prometheus_javaagent-0.3.1.jar",
+                            "port": 8090}}}
```

Comment:
Should these (`jmxExporterJar`, `port`) be set in the configuration or validation rules?